### DISCUSSION TITLE
Emails - Downgrade/hide "Duplicate of transaction"

### DISCUSSION THREAD

**Discussion by scottgrobinson** [firefly-iii/firefly-iii#9222] [Firefly III]
I'm importing from gocardless and get an email every day with hundreds of errors similar to 

```
Line #1: transactions.0.description: Duplicate of transaction #481. (original value: "INT'L 1276585576 [APPLE.COM/BILL](http://apple.com/BILL) [APPLE.COM/BIL](http://apple.com/BIL)")
```

I suppose it is an error because it's a duplicate transaction, but is there any way to "hide" these from the email so I only receive an email if there is "real" error?

**Comment by JC5** [comment]
Yes, check out the .env.example file.

**Comment by scottgrobinson** [comment]
Thanks. For anyone else coming across this it's "IGNORE_DUPLICATE_ERRORS"

For what it's worth it may be moving that or re-labelling that so it's clear it's specific to emails (something like MAIL_REPORT_IGNORE_DUPLICATE_ERRORS) but that's just one persons opinion :) Keep up the good work here as always, thank you for the great software :)
